OpenRA.Mods.Common/Traits/TransformOnCapture.cs

```diff
@@ -20,6 +20,9 @@ public class TransformOnCaptureInfo : ITraitInfo
 		public readonly int ForceHealthPercentage = 0;
 		public readonly bool SkipMakeAnims = true;
 
+		[Desc("Triggered only capturer's CaptureType contains this.")]
+		public readonly string Type = "husk";
```

Comment:
Can we please default this to `null`, and then have `IsValidCaptor` immediately return true if `string.IsNullOrEmpty(info.Type)`?  This will then maintain the default behaviour for use cases that don't need or want to filter on capture type.